client/src/includes/initIconSprite.test.js

```diff
@@ -0,0 +1,61 @@
+import { initIconSprite } from './initIconSprite';
+
+const flushPromises = () => new Promise(setImmediate);
+describe('initIconSprite', () => {
```

Comment:
```suggestion

describe('initIconSprite', () => {
```

Same here, just a line break above the tests starting and it would make this easier to read.